types/unordered_set_type.toml

```diff
@@ -67,3 +67,78 @@ struct TypeHandler<DB, %1%<T0, T1, T2, T3>> {
   }
 };
 """
+
+traversal_func = """
+auto tail = returnArg
+  .write((uintptr_t)&container)
+  .write(container.bucket_count())
+  .write(container.size());
+
+for (const auto &it : container) {
+  tail = tail.delegate([&it](auto ret) {
+    return OIInternal::getSizeType<DB>(it, ret);
+  });
+}
+
+return tail.finish();
+"""
+
+[[codegen.processor]]
+type = "types::st::VarInt<DB>"
+func = """
+el.pointer = std::get<ParsedData::VarInt>(d.val).value;
+"""
+
+[[codegen.processor]]
+type = "types::st::VarInt<DB>"
+func = """
+// Using the container's capacity to temporarily store the number of buckets
+// TODO: Is there another way to pass a value across processors?
```

Comment:
I think this is a smart use of what's there for now. The current alternative is to make the next processor take a pair, but as discussed it's messy in the current implementation.
Future options:
- Make the interface for taking out of a complex type in a processor better (my preferred).
- Add an option for naming variables like `VarInt` and making them available in future processors (complex to add to the current implementation of separate functions).
- Having some sort of dump space where variables can be carried through (complex to manage).